backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -652,6 +660,9 @@ public static Bundle createFhirBundle(
                     .setFullUrl(pair.getFirst())
                     .setResource(pair.getSecond())));
 
+    bundle.setMeta(
+        new Meta()
+            .addTag(PROCESSING_ID_SYSTEM, processingId, PROCESSING_ID_DISPLAY.get(processingId)));
```

Comment:
I think the preference here is to use the library's built in auto-creation when you call `get` on most things. 
```java
bundle.getMeta().addTag(PROCESSING_ID_SYSTEM, processingId, PROCESSING_ID_DISPLAY.get(processingId)));
```

Reply:
Interesting, so the library should be able to handle adding a new Meta object if not initialized. Will make the change

Reply:
It will yes! I think in the converter you can see the first methods that were written did not take into consideration so that's why they do something similar here. 

In [their documentation](https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-structures-r4/org/hl7/fhir/r4/model/Configuration.html) for configuration, they would like people to use the auto creation and use the `hasX()` method to check if a value is set.

Reply:
thats awesome!
![image](https://user-images.githubusercontent.com/4952042/219497151-8a1b89e2-e930-4712-9631-2faeab3ea40e.png)